src/Algorithms/ASB07/reach_homog.jl

```diff
@@ -9,9 +9,10 @@ function reach_homog_ASB07!(F::Vector{ReachSet{N, Zonotope{N, VN, MN}}},
                             max_order::Integer,
                             X::Universe,
                             recursive::Val{true},
-                            reduction_method::AbstractReductionMethod) where {N, VN, MN}
+                            reduction_method::AbstractReductionMethod,
+                            t0) where {N, VN, MN}
     # initial reach set
-    Δt = zero(N) .. δ
+    Δt = t0 .. δ
```

Comment:
Should it not be `t0 .. t0 + δ`?

Reply:
yes, thanks.

fwiw this is an "ad-hoc" time shift to be used internally, so that we don't have to make extra copy operations for hybrid systems, or to add wrapper types just to shift the solution in time. it does not close https://github.com/JuliaReach/ReachabilityAnalysis.jl/issues/21 because that is something else: in that case one wants to solve `x'(t) = f(x(t))` for any time interval `[t1, t2]` (this feature is alreay available in TMJets however).